platform/core/src/services/ViewportGridService/ViewportGridService.ts

```diff
@@ -38,9 +37,6 @@ class ViewportGridService extends PubSubService {
     if (setActiveViewportIndexImplementation) {
       this.serviceImplementation._setActiveViewportIndex = setActiveViewportIndexImplementation;
     }
-    if (setDisplaySetsForViewportImplementation) {
-      this.serviceImplementation._setDisplaySetsForViewport = setDisplaySetsForViewportImplementation;
```

Comment:
For sake of backward compatibility I would say to not remove this. Instead we just factory to the new reducer you have created.

Also, if we decide to really change this we need to ensure the usage of it is also updated (cornerstone-dicom-seg, cornerstone-dicom-sr)

Reply:
It is backwards compatible - I added a function that supports that directly, but I'm really against having two functions to call in the back end as it makes it harder to figure out what is happening.  We are still pre-release on v3-stable so don't have to absolutely worry about backwards compatibility.